src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -480,7 +444,9 @@ const WorkflowView = _.flow(
         !isRedacted ? filterConfigIO(inputsOutputs) : _.identity
       )(config)
 
-      const selectedSnapshotEntityMetadata = modifiedConfig.dataReferenceName ? await Ajax(signal).Workspaces.workspace(namespace, name).snapshotEntityMetadata(namespace, modifiedConfig.dataReferenceName) : undefined
+      const selectedSnapshotEntityMetadata = modifiedConfig.dataReferenceName ?
```

Comment:
From here down is just Intellij breaking up long lines, which is long overdue for this file, with the exception of the newly renamed `DocumentationCollapse` on line 658.